data/templates/ipsec/swanctl/l2tp.j2

```diff
@@ -8,6 +8,9 @@
         dpd_timeout = 45s
         rekey_time = {{ l2tp_ike.lifetime if l2tp_ike else l2tp.ike_lifetime }}s
         reauth_time = 0
+{% if uniqreqids %}
```

Comment:
Any reason to not use the common pattern

```suggestion
{% if uniqreqids is vyos_defined %}
```

Which also tests if the supplied variable is not `None`?